.circleci/config.yml

```diff
@@ -346,14 +346,25 @@ jobs:
             cd ~/adapter-deps/install/bin/
             wget https://dl.min.io/server/minio/release/linux-amd64/minio
             chmod +x minio
+      - run:
+          name: "Install Hadoop dependency"
+          command: |
+            set -xu
+            cd ~/adapter-deps/install/bin/
+            wget https://archive.apache.org/dist/hadoop/common/hadoop-2.10.1/hadoop-2.10.1.tar.gz
+            tar xf hadoop-2.10.1.tar.gz
+            chmod +x hadoop-2.10.1
+            cp -a hadoop-2.10.1 /usr/local/
+            yum install java-1.8.0-openjdk
```

Comment:
Should I put installing java as part of the circle ci image instead? into the setup circleci script.
If I do that how do I generate a new image?